go-controller/vendor/k8s.io/kubelet/pkg/apis/podresources/v1/api.proto

```
@@ -0,0 +1,121 @@
+// To regenerate api.pb.go run `hack/update-codegen.sh protobindings`
+syntax = "proto3";
+
+package v1;
+option go_package = "k8s.io/kubelet/pkg/apis/podresources/v1";
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
```

Buf (1.55.1)

import "github.com/gogo/protobuf/gogoproto/gogo.proto": file does not exist

(COMPILE)